roles/ansible_tower/tasks/main.yaml

```diff
@@ -0,0 +1,38 @@
+---
+- name: install some useful packages
+  yum: name={{ item }}
+  with_items:
+  - git
+  - python-pip
+  - unzip
+  - python-psphere
+  - ansible
+  - telnet
+  - ack
+
+- name: download Tower setup
+#  get_url: url=http://releases.ansible.com/ansible-tower/setup/ansible-tower-setup-{{tower_version}}.tar.gz
+  get_url: url=http://releases.ansible.com/ansible-tower/setup/ansible-tower-setup-2.1.1.tar.gz
+    dest=/opt/ force=yes
```

Comment:
This is going to run every single config run. Do we really need it to download each time?

Consider using a conditional, like if that file doesn't exist then download it, or something like that.